bundles/binding/org.openhab.binding.caldav-command/src/main/java/org/openhab/binding/caldav_command/internal/CalDavBinding.java

```diff
@@ -0,0 +1,509 @@
+	
+	private List<String> disabledItems = new ArrayList<String>();
+	
+	public CalDavBinding() {
+	}
+	
+	public void setItemRegistry(ItemRegistry itemRegistry) {
+		this.itemRegistry = itemRegistry;
+	}
+	
+	public void unsetItemRegistry(ItemRegistry itemRegistry) {
+		this.itemRegistry = null;
+	}
+	
+	public void setCalDavLoader(CalDavLoader calDavLoader) {
+		this.calDavLoader = calDavLoader;
+		this.calDavLoader.addListener(this);
+	}
+	
+	public void unsetCalDavLoader() {
+		this.calDavLoader.removeListener(this);
+		this.calDavLoader = null;
+	}
+	
+	public void activate() { }
+	
+	public void deactivate() { }
+
+	
+	@Override
+	public void updated(Dictionary<String, ?> properties)
+			throws ConfigurationException {
+		if (properties != null) {
+			logger.debug("reading configuration data...");
+			String read = (String) properties.get(KEY_READ_CALENDARS);
+			if (read != null) {
+				for (String value : read.split(",")) {
+					this.readCalendars.add(value.trim());
+				}
+			}
+		}
+	}
+	
+	private String parseContent(String content, String scope) {
+		try {
+			StringBuilder out = new StringBuilder();
+			BufferedReader reader = new BufferedReader(new StringReader(content));
+		
+			String line = null;
+			while ((line = reader.readLine()) != null) {
+				line = line.trim();
+				
+				if (line.startsWith(scope)) {
+					if (out.length() > 0) {
+						out.append(",");
+					}
+					out.append(line.substring(scope.length() + 1));
+				}
+			}
+			return out.toString();
+		} catch (IOException e) {
+			logger.error("cannot parse event content", e);
+			return "";
+		}
+	}
+	
+	private CalDavBindingProvider getCalDavBindingProvider() {
+		for (CalDavBindingProvider provider : providers) {
+			return provider;
+		}
+		return null;
+	}
+	
+	private void handleForEventMap(CalDavEvent event, String scope, boolean add) {
+		if (!readCalendars.contains(event.getCalendarId())) {
+			logger.trace("event '{}' is not in used calendars", event.getShortName());
+			return;
+		}
+		
+		if (event.getContent() == null) {
+			logger.warn("no content for event: {}", event.getShortName());
+			return;
+		}
+		
+		String content = parseContent(event.getContent(), scope);
+		
+		if (content.trim().isEmpty()) {
+			logger.warn("no content for event: {}", event.getShortName());
+			return;
+		}
+		
+		String[] commands = content.split(",");
+		for (String itemCommand : commands) {
+			String[] commandSplit = itemCommand.split(":");
+			String itemName = commandSplit[0];
+			String commandString = commandSplit[1];
+			
+			Item item = null;
+			try {
+				item = itemRegistry.getItem(itemName);
+			} catch (ItemNotFoundException e) {
+				logger.error("item '" + itemName + "' could not be found");
+				return;
+			}
+			if (item == null) {
+				logger.error("cannot find item: {}", itemName);
+				return;
+			}
```

Comment:
this if will never be true because in case no item is found the try-catch above leads to a return already